src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/ProjectSystemProviderContext.cs

```diff
@@ -12,15 +13,41 @@ namespace NuGet.PackageManagement.VisualStudio
     /// </summary>
     public class ProjectSystemProviderContext
     {
+        public const string RESTORE_PROJECT_STYLE = "RestoreProjectStyle";
```

Comment:
Use camel casing, all caps for const is pretty common, but I most other consts in the project are camel cased